### ISSUE TITLE
No module named 'pyldapi' 

### ISSUE THREAD

**Issue by MFSY** [RDFLib/pySHACL#3]
Hi,
This validator arrived just at the right time to enable more adoption of SHACL. Thank you for this effort.
I'm using a jupyter notebook running on python 3.6 and I installed the pyshacl module with:

!pip install git+https://github.com/RDFLib/pySHACL@v0.8.0#egg=pyshacl

As suggested in the 'Use' section of the [README file](https://github.com/RDFLib/pySHACL/blob/master/README.md), I tried a basic validation by running:

```
from pyldapi import validate
validate(target_graph, shacl_graph, inference='rdfs', abort_on_error=False)
```

But I got a **ModuleNotFoundError: No module named 'pyldapi'**

I guess it's because the validate function seems to be part of the pyshacl module. I got it right by running :

```
from pyshacl import validate 
validate(target_graph, shacl_graph, inference='rdfs', abort_on_error=False)
```

Thanks.

**Comment by ashleysommer** [comment]
Woops, that's my mistake. I copy & pasted the example code in the README from a different project, and forgot to change the module name. Thanks for the PR.

**Comment by ashleysommer** [comment]
Fixed by #4
